packages/fast-components-styles-msft/src/progress/index.ts

```diff
@@ -1,22 +1,23 @@
 import designSystemDefaults, { IDesignSystem, withDesignSystemDefaults } from "../design-system";
-import { ComponentStyles, ComponentStyleSheet, ICSSRules } from "@microsoft/fast-jss-manager";
+import { ComponentStyles, ComponentStyleSheet } from "@microsoft/fast-jss-manager";
 import { IMSFTProgressClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
 import { ensureBrandNormal, largeContrast } from "../utilities/colors";
 import { get } from "lodash";
+import { toPx } from "@microsoft/fast-jss-utilities";
 
-/* tslint:disable-next-line */
-const styles: ComponentStyles<IMSFTProgressClassNameContract, IDesignSystem> = (config: IDesignSystem): ComponentStyleSheet<IMSFTProgressClassNameContract, IDesignSystem> => {
+const styles: ComponentStyles<IMSFTProgressClassNameContract, IDesignSystem> = (
+    config: IDesignSystem
+): ComponentStyleSheet<IMSFTProgressClassNameContract, IDesignSystem> => {
     const designSystem: IDesignSystem = withDesignSystemDefaults(config);
     const brandColor: string = ensureBrandNormal(config);
-    /* tslint:disable-next-line */
     const determinateBackgroundColor: string = largeContrast(designSystem.contrast, designSystem.backgroundColor, brandColor);
 
     return {
         progress: {
             display: "flex",
             width: "100%",
             alignItems: "center",
-            height: "10px",
+            height: toPx(designSystem.designUnit),

```

Comment:
this is changing to 4px, is this right?

Reply:
yes. The vertical height of the visible UI is 4px, however this was set to ten. The content-box should match the physical height of the UI element. If we want to add vertical spacing we should use margin.